drivers/include/periph/rtt.h

```diff
@@ -32,6 +32,39 @@ extern "C" {
 /* guard file in case no RTT device was specified */
 #if RTT_NUMOF
 
+#ifndef RTT_FREQUENCY
+#warning "RTT_FREQUENCY undefined. Set RTT_FREQUENCY to the number of ticks" \
+         "per second for the current architecture."
+#endif
+
+/**
+ * @brief       Convert microseconds to rtt ticks
+ * @param[in]   us number of microseconds
+ * @return      rtt ticks
+ */
+#define RTT_US_TO_TICKS(us)     ((uint32_t)((uint64_t)(us) * RTT_FREQUENCY / 1000000UL))
+
+/**
+ * @brief       Convert milliseconds to rtt ticks
+ * @param[in]   ms number of milliseconds
+ * @return      rtt ticks
+ */
+#define RTT_MS_TO_TICKS(ms)     ( RTT_US_TO_TICKS((ms) * 1000) )
```

Comment:
Could you please provide an example? Calculation is done from left to right and the bigger the value before division, the more stable it is, isn't it? Please note that ` * 1000` is done inside macro argument